cmd/skywire-cli/commands/visor/gen-config.go

```diff
@@ -147,7 +147,7 @@ func defaultConfig() *visor.Config {
 
 	conf.Interfaces.RPCAddress = "localhost:3435"
 
-	conf.AppServerSockFile = "/tmp/visor_" + pk.Hex() + ".sock"
+	conf.AppServerAddr = ":5505"
```

Comment:
Should we bind to `0.0.0.0` instead of `localhost` here? Probably it's also worth extracting a constant with the port as this magic number is widely used in the codebase.